package.json

```diff
@@ -165,7 +168,7 @@
     "virtual"
   ],
   "lint-staged": {
-    "*.(js|json)": [
+    "*.{ts,tsx,js,json}": [
```

Comment:
previous glob didn't catch files properly. new one is applied on everything code related , even TS files -> consistency FTW